site2/website/release-notes.md

```diff
@@ -1,11 +1,180 @@
 
 ## Apache Pulsar Release Notes
 
-### 2.7.0 &mdash; Not Yet Released <a id=“2.7.0”></a>
+### 2.7.0 &mdash; 2020-11-25 <a id=“2.7.0”></a>
+
+The following lists fixes and enhancements in the 2.7.0 release.
+
+#### Transactions
+
+- Implementation of Transaction Buffer Client [#6544](https://github.com/apache/pulsar/pull/6544)
+- Support produce messages with transaction and commit transaction [#7552](https://github.com/apache/pulsar/pull/7552)
+- Support consume transaction messages [#7781](https://github.com/apache/pulsar/pull/7781) [#7833](https://github.com/apache/pulsar/pull/7833)
+- Message acknowledgment with transaction [#7856](https://github.com/apache/pulsar/pull/7856) [#8007](https://github.com/apache/pulsar/pull/8007)
+- Support transaction abort on partition [#7953](https://github.com/apache/pulsar/pull/7953)
+- Support transaction abort on subscription [#7979](https://github.com/apache/pulsar/pull/7979)
+- Handle pending ack at the client side [#8037](https://github.com/apache/pulsar/pull/8037)
+- Pending ack state implementation [#8426](https://github.com/apache/pulsar/pull/8426)
+- Support get reponse for message acknowledge [#8161](https://github.com/apache/pulsar/pull/8161)
+- Refactor the transaction buffer implementation [#8291](https://github.com/apache/pulsar/pull/8291) [#8347](https://github.com/apache/pulsar/pull/8347)
+- Transaction marker deletion [#8318](https://github.com/apache/pulsar/pull/8318)
+- Support produce messages with transaction in batch [#8415](https://github.com/apache/pulsar/pull/8415)
+- Register transaction metadata before send or ack messages [#8493](https://github.com/apache/pulsar/pull/8493)
+- Expose transaction interface [#8505](https://github.com/apache/pulsar/pull/8505)
+- Guarantee transaction metadata handlers connected [#8563](https://github.com/apache/pulsar/pull/8563)
+- Add the batch size in transaction ack command [#8659](https://github.com/apache/pulsar/pull/8659)
+- Transaction Log implementation [#8658](https://github.com/apache/pulsar/pull/8658)
+
+#### Topic policy
+
+- Support set message TTL on topic level [#7738](https://github.com/apache/pulsar/pull/7738)
```

Comment:
support + doing, so change "support set" to "support setting", same comment for the below.